src/platform.deno.ts

```diff
@@ -69,12 +70,22 @@ export class InputFile {
             | AsyncIterable<Uint8Array>,
         filename?: string,
     ) {
-        this[inputFileData] = file;
+        this.fileData = file;
         if (filename === undefined && typeof file === "string") {
             filename = basename(file);
         }
         this.filename = filename;
     }
+    get [inputFileData]() {
+        if (this.consumed) {
+            throw new Error("Cannot reuse InputFile data source!");
+        }
+        const data = this.fileData;
+        if (typeof data !== "string" && (!(data instanceof Uint8Array))) {
+            this.consumed = false;
```

Comment:
Probably `true`? Or I don't get the point

Reply:
Thanks, I renamed the flag from `consumable` to `consumed` and that change slipped through.